src/Seksjon/Sertifikat.elm

```diff
@@ -0,0 +1,1323 @@
+                            , lagtTilSpørsmålCmd model.debugStatus
+                            )
+                                |> IkkeFerdig
+
+                        Err error ->
+                            ( Model model, logFeilmelding error "Hent SertifikatTypeahead" )
+                                |> IkkeFerdig
+
+                EndreOpplysninger skjema ->
+                    case result of
+                        Ok suggestions ->
+                            ( TypeaheadState.updateSuggestions "" (List.take 10 suggestions)
+                                |> SertifikatSkjema.mapTypeaheadState skjema
+                                |> EndreOpplysninger
+                                |> oppdaterSamtaleSteg model
+                            , lagtTilSpørsmålCmd model.debugStatus
+                            )
+                                |> IkkeFerdig
+
+                        Err error ->
+                            ( Model model, logFeilmelding error "Hent SertifikatTypeahead" )
+                                |> IkkeFerdig
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        HovrerOverTypeaheadSuggestion typeahead ->
+            case model.aktivSamtale of
+                RegistrerSertifikatFelt typeaheadState ->
+                    ( Model
+                        { model
+                            | aktivSamtale =
+                                typeaheadState
+                                    |> TypeaheadState.updateActive typeahead
+                                    |> RegistrerSertifikatFelt
+                        }
+                    , Cmd.none
+                    )
+                        |> IkkeFerdig
+
+                _ ->
+                    ( Model model, Cmd.none )
+                        |> IkkeFerdig
+
+        TrykkerTypeaheadTast operation ->
+            case model.aktivSamtale of
+                RegistrerSertifikatFelt typeaheadState ->
+                    case operation of
+                        Typeahead.ArrowUp ->
+                            ( Model
+                                { model
+                                    | aktivSamtale =
+                                        typeaheadState
+                                            |> TypeaheadState.arrowUp
+                                            |> RegistrerSertifikatFelt
+                                }
+                            , Cmd.none
+                            )
+                                |> IkkeFerdig
+
+                        Typeahead.ArrowDown ->
+                            ( Model
+                                { model
+                                    | aktivSamtale =
+                                        typeaheadState
+                                            |> TypeaheadState.arrowDown
+                                            |> RegistrerSertifikatFelt
+                                }
```

Comment:
Og her. Er det gjort sånn her i en annen modul som du har basert deg på? For i så fall kan vi godt endre det i den modulen også.

Reply:
:D ja, har stort sett basert meg på arbeidserfaring og fagdokumentasjon, ser ut som begge de gjør sånn i blant :)